packages/server/graphql/public/types/User.ts

```diff
@@ -28,6 +28,20 @@ import {UserResolvers} from '../resolverTypes'
 
 declare const __PRODUCTION__: string
 
+const MODEL = 'Embeddings_ember_1'
+const EMBED_URL = (() => {
+  try {
+    const availableModels =
+      process.env.AI_EMBEDDING_MODELS && JSON.parse(process.env.AI_EMBEDDING_MODELS)
+    return availableModels.find(
+      ({model}: {model?: string}) => model?.split(':')[1] === 'llmrails/ember-v1'
```

Comment:
@Dschoordsch why is `llmrails/ember-v1` hardcoded? That's just a possibility but in the future we might have other thing there. I might be lacking of JS knowledge to read this expression 😺

Reply:
You're right, it's hardcoded. It's also hardcoded here: https://github.com/ParabolInc/parabol/blob/46770a048bfd4cd21c0e1cab89abdc01f8a79af3/packages/embedder/ai_models/TextEmbeddingsInference.ts#L14

I tried to strike the balance between getting stuff out now vs. making it future proof. Even if we add more models, we would need to test which one to use here and adjust accordingly. Having it configurable via environment for example add some complexity which might never be needed.

Reply:
If I'm not mistaking, on that file TextEmbeddingsInference.ts you have that model but there is [another one there too](https://github.com/ParabolInc/parabol/blob/46770a048bfd4cd21c0e1cab89abdc01f8a79af3/packages/embedder/ai_models/TextEmbeddingsInference.ts#L8).

Reply:
If I'm not mistaking, on that file TextEmbeddingsInference.ts you have that model but there is [another one there too](https://github.com/ParabolInc/parabol/blob/46770a048bfd4cd21c0e1cab89abdc01f8a79af3/packages/embedder/ai_models/TextEmbeddingsInference.ts#L8).